trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
+  end
```

Comment:
- [ ]  Please make sure to write the correct method for the correct name in the trimmer class.
The name has a maximum of 10 characters. If it's longer it should trim the word.